src/pages/api/v1/job-descriptions/index.ts

```diff
@@ -0,0 +1,45 @@
+import prisma from '../../../../lib/prisma';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const getJobDescriptions = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+) => {
+  try {
+    const data = await prisma.jobDescription.findMany();
+    res.status(200).json(data);
+  } catch (error) {
+    res.status(500).json({
+      message:
+        'Something went wrong on our side and we failed to get Job Descriptions.',
+    });
+  }
+  if (req.method === 'POST') {
+  }
+};
+
+const postHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const { rawText } = req.body;
+    console.log(rawText);
```

Comment:
Looks like this was accidentally left in. Can you please remove this console.log?